### DISCUSSION TITLE
Vérifier puis geler les interfaces en vu d'une release

### DISCUSSION THREAD

**Discussion by aksiome** [Gunivers/Bookshelf#164] [General]
The goal of this post is to list and discuss problematic interfaces before being stuck with them. The following list will get updated when changes are converted to issues or on new ideas...

### Awaiting discussion

- `bs.view` can_see_ata and in_view_ata (macro vs simplicity with side effects)

- `bs.raycast` & `bs.view`: define raycast related functions in raycast module and add an optional alias (not a dependency) to the view module

### Done but not discussed yet

- `bs.move` use the biggest hitbox between the entity and all its passengers. This way we can for example easily add an hitbox to display_entities.

**Comment by theogiraudet** [comment]
Pour les "à discuter", je n'ai pas spécialement d'avis ne connaissant pas trop les modules concernés…
Pour le premier, je suppose que plus on peut gérer de cas mieux c'est, même si c'est légèrement au détriment des performances.
Pour les "à confirmer", les deux premiers me vont. Je ne suis pas sûr d'avoir compris le troisième.
